Mage.Sets/src/mage/cards/v/ValiantRescuer.java

```diff
@@ -0,0 +1,48 @@
+package mage.cards.v;
+
+import java.util.UUID;
+
+import mage.MageInt;
+import mage.abilities.common.CycleControllerTriggeredAbility;
+import mage.abilities.costs.mana.ManaCostsImpl;
+import mage.abilities.effects.common.CreateTokenEffect;
+import mage.abilities.keyword.CyclingAbility;
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.SubType;
+import mage.game.permanent.token.HumanSoldierToken;
+
+/**
+ * 
+ * @author Emigara
+ *
+ */
+public class ValiantRescuer extends CardImpl{
+
+	public ValiantRescuer(UUID ownerId, CardSetInfo setInfo) {
+		super(ownerId, setInfo, new CardType[] {CardType.CREATURE}, "{1}{W}");
+		
+		this.subtype.add(SubType.HUMAN);
+		this.subtype.add(SubType.SOLDIER);
+		
+		this.power = new MageInt(3);
+		this.toughness = new MageInt(1);
+		
+		this.addAbility(new CycleControllerTriggeredAbility(
```

Comment:
This needs to trigger only the first time something cycles in a turn.

Reply:
is there a similar card with a first time int he turn condition on it so i can look and learn how to implement it?

Reply:
Take a look at cards like [these](https://scryfall.com/search?q=o%3A%22for+the+first+time%22), it's sort of complicated which is why I hadn't gotten to it yet.

Reply:
got it from the surveil first time in a turn card.  godda make a watcher nd use that to check the condition